src/vs/workbench/contrib/notebook/browser/controller/editActions.ts

```diff
@@ -621,3 +625,62 @@ registerAction2(class SelectNotebookIndentation extends NotebookAction {
 		return;
 	}
 });
+
+registerAction2(class CommentSelectedCellsAction extends NotebookMultiCellAction {
+	constructor() {
+		super({
+			id: COMMENT_SELECTED_CELLS_ID,
+			title: localize('commentSelectedCells', "Comment Selected Cells"),
```

Comment:
Since it actually toggles the commenting state, would it make sense to include "Toggle" in the command title?